contracts/p1/RToken.sol

```diff
@@ -443,6 +444,38 @@ contract RTokenP1 is ComponentP1, ERC20PermitUpgradeable, IRToken {
 
     /// @custom:governance
     function setIssuanceThrottleParams(ThrottleLib.Params calldata params) public governance {
+        _setIssuanceThrottleParams(params);
+        require(
+            isRedemptionThrottleGreaterByDelta(params, redemptionThrottle.params),
+            "redemption throttle too low"
+        );
+    }
+
+    /// @custom:governance
+    function setRedemptionThrottleParams(ThrottleLib.Params calldata params) public governance {
+        _setRedemptionThrottleParams(params);
+        require(
+            isRedemptionThrottleGreaterByDelta(issuanceThrottle.params, params),
+            "redemption throttle too low"
+        );
+    }
+
+    /// @custom:governance
+    function setThrottleParams(
+        ThrottleLib.Params calldata issuanceParams,
+        ThrottleLib.Params calldata redemptionParams
+    ) external governance {
+        _setIssuanceThrottleParams(issuanceParams);
+        _setRedemptionThrottleParams(redemptionParams);
+        require(
+            isRedemptionThrottleGreaterByDelta(issuanceParams, redemptionParams),
+            "redemption throttle too low"
+        );
+    }
+
+    // === Private Helpers ===
+
+    function _setIssuanceThrottleParams(ThrottleLib.Params calldata params) public governance {
```

Comment:
duplicate `governance` modifiers on the internal functions. I don't have much of a preference for which to keep, I guess if I _had_ to pick I would go external